presidio-analyzer/tests/conftest.py

```diff
@@ -52,6 +52,15 @@ def nlp_engines(request, nlp_engine_provider) -> Dict[str, NlpEngine]:
     for name, engine_cls in nlp_engines.items():
         if name == "spacy" and not request.config.getoption("--runfast"):
             available_engines[f"{name}_en"] = engine_cls({"en": "en_core_web_lg"})
+        elif name == "transformers" and not request.config.getoption("--runfast"):
+            available_engines[f"{name}_en"] = engine_cls(
+                {
+                    "en": {
+                        "spacy": "en_core_web_lg",
```

Comment:
Shouldn't it be a spacy transformer model as well? `en_core_web_trf`?

Reply:
The `trf` models are standard spaCy models, so the `huggingface` option is not required for them. This PR would handle non-spaCy transformers models.

Reply:
Also, the spaCy model here is only used as a pipeline, not as a NER model. See here: https://github.com/microsoft/presidio/issues/887